crates/sui-framework/sources/governance/sui_system.move

```diff
@@ -433,6 +439,10 @@ module sui::sui_system {
         stake_subsidy_rate: u64, // what percentage of the total stake do we mint as stake subsidy.
         ctx: &mut TxContext,
     ) {
+        if (self.safe_mode) {
+            return advance_epoch_safe_mode(self, new_epoch, ctx)
+        };
```

Comment:
We need a way to set `self.safe_mode` back to `false` too?

Reply:
I don't think so, unless we support framework upgrade

Reply:
The goal of this PR isn't to allow us recover and be able to continue doing tokenomics experiments, but rather just keep the network running.

Reply:
Makes sense